packages/service/src/utils/users.ts

```diff
@@ -1,5 +1,26 @@
+import { upsert } from "zapatos/db";
 import { db, pool, schema } from "../db";
 
 export function createUser(props: { name: string }) {
-  return db.upsert("users", props, "name").run(pool);
+  // return db.upsert("users", props, "name").run(pool);
+  return db.sql<
+    schema.users.SQL,
+    schema.users.Selectable
+  >`INSERT INTO ${"users"} (${"name"}) VALUES (${{ name: props.name }})`.run(
+    pool
+  );
 }
+
+export const selectUsers = () => {
+  const users = db.sql<
+    schema.users.SQL,
+    schema.users.Selectable[]
+  >`SELECT * FROM ${"users"}`.run(pool);
+  return users;
+};
+
+export const createPswash = (user: { password: string; id: string }) => {
+  return db.sql`UPDATE ${"users"} SET pswhash = crypt(${"password"}, gen_salt('md5')) WHERE ${{
+    id: user.id,
```

Comment:
"password" is not a column yet. That will be added in a future commit through a migration update.

Reply:
We should probably add the password column **before** making these changes